examples/async-with-routing/components/Layout.js

```diff
@@ -0,0 +1,11 @@
+import React, { Component } from 'react'
+
+export default class Layout extends Component {
```

Comment:
`Layout` is confusing. I don’t mind the component per se but right now it looks pointless. If you can use it to make the page look nicer in some way, go for it, but an empty component that just renders its children is not very useful.